temp_library/classes.py

```diff
@@ -1,13 +1,30 @@
 import temp_library
+import random
 
+class Decision(object):
+    def __init__(self, next__scene, successRate=1.0):
+        self.next_scene = next__scene
+        self.successRate = successRate
+
+    def GetScene(self):
+        return self.next_scene
+
+    def GetSceneName(self):
+        return self.next_scene.getName()
+
+    def GetSuccessRate(self):
+        return self.successRate
 
-# class Decision:
 
 class Scene(object):
-    def __init__(self, desc, choice, decision):
+    def __init__(self, name, desc, choice):
+        self.name = name
         self.desc = desc
         self.choice = choice
-        self.decision = decision
+        
+
+    def GetName(self):

```

Comment:
Try to use a python linter (python uses snake_case not CamelCase)

Reply:
Search for pylint/black they help linting (fixing code styles).

Reply:
or flake8

Reply:
i do use pylint but it did not say anything about the CamelCase

Reply:
i will change it anyway